src/components/ReportActionItem/IOUAction.js

```diff
@@ -30,9 +34,21 @@ const propTypes = {
         hasOutstandingIOU: PropTypes.bool.isRequired,
     }),
 
+    /** IOU report data object */
+    iouReport: PropTypes.shape({
```

Comment:
qq: We have `iouReportPropTypes` in `pages/iouReportPropTypes`. Any specific reason we're creating a new `PropTypes.shape` here?

Reply:
I think I copied this from `IOUAction` and the good thing is that is shows just what we needs to use from the object, in this case only `currency` is needed. But I'm happy to change this.